hca/staging_area_validator.py

```diff
@@ -372,12 +383,15 @@ def validate_json(cls, file_json: JSON, schema: Optional[JSON] = None) -> None:
     @classmethod
     # setting to maxsize=None so as not to evict old values, and maybe help avoid connectivity issues (DI-22)
     @lru_cache(maxsize=None)
-    def _download_schema(cls, schema_url: str) -> JSON:
+    def _download_schema(cls, schema_url: str, total_retries: int) -> JSON:
         log.debug("Downloading schema %s", schema_url)
 
         s = requests.Session()
+        print(f"total_retries = {total_retries}")
```

Comment:
This causes noisy output (`total_retries = 10` printed before every  file validated). Consider changing the `print` to a `log.debug`, or simply removing it.